src/Components/Users/ManageUsers.tsx

```diff
@@ -347,10 +347,15 @@ export default function ManageUsers() {
                   )}
                   <div className="flex-shrink-0 text-sm text-gray-600 min-width-50">
                     {user.last_login && cur_online ? (
-                      <span>Currently Online</span>
+                      <span>
+                        {" "}
+                        <i className="fa-solid fa-clock"></i> Currently Online
+                      </span>
```

Comment:
You can simply do
```suggestion
                      <span>
                        {" "}
                        <i className="fa-solid fa-clock"></i> {user.last_login && cur_online ? "Currenlty online" : "Last Online"}
                      </span>
```

Reply:
@developedBySJ if I add the condition here and remove the below one then it will never trigger "Last online". It will check
only when ``` user.last_login && cur_online``` in line 349 is true when it is false i will not go there.